token/services/auditor/auditor.go

```diff
@@ -146,3 +147,35 @@ func (a *Auditor) GetTokenRequest(txID string) ([]byte, error) {
 func (a *Auditor) Check(context context.Context) ([]string, error) {
 	return a.checkService.Check(context)
 }
+
+func (a *Auditor) completeInputsWithEmptyEID(record *token.AuditRecord) error {
+	filter := record.Inputs.ByEnrollmentID("")
```

Comment:
Why would an input have an empty enrollment id?

Reply:
When redeeming tokens during an issue. These tokens might not be parsable by the current token driver.